apps/guardian-ui/package.json

```diff
@@ -23,10 +23,13 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-icons": "^4.7.1",
+    "react-qr-code": "^2.0.12",
```

Comment:
This seems like a decently popular library, although relies it on `qr.js` which appears to no longer be maintained (last update 10 years ago) and is hard to track down the source code for. But swapping QR implementations is pretty trivial, so I'm not worried about what's chosen as long as it doesn't look like a security risk.